src/AaveV3ArbNativeUSDCListing_20230621/DeployUSDCListing.s.sol

```diff
@@ -20,10 +20,7 @@ contract USDCNPayloadProposal is EthereumScript {
       payloads,
       GovHelpers.ipfsHashFile(
         vm,
-        'src/AaveV3ArbNativeUSDCListing_20230621/ADD-NATIVE-USDC-ARB-V3-AIP.md',
-        // if you set `upload` to `true`, your env must contain PINATA_KEY & PINATA_SECRET
```

Comment:
we had this here as an example mostly for ppl who don't want to use `aave-proposals` repo.
If you use `aave-proposals` repo just omit the last parameter plz